<ui version="4.0" >
 <class>TUview</class>
 <widget class="QWidget" name="TUview" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>13</hsizetype>
    <vsizetype>13</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777187</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Detail</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="statusTip" >
   <string/>
  </property>
  <property name="whatsThis" >
   <string>&lt;h3>Source and Target View&lt;/h3>It contains source of unit in the upper text box, and target of unit in the below text box. If unit is plural, it will display multi of tabs.</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QStackedWidget" name="sourceStacked" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="txtSource" >
          <property name="toolTip" >
           <string>Original Source String</string>
          </property>
          <property name="statusTip" >
           <string/>
          </property>
          <property name="whatsThis" >
           <string>&lt;h3>Original Source String&lt;/h3>This part of the window shows you the original string of the currently displayed entry. &lt;br>You can not edit this string.</string>
          </property>
          <property name="tabChangesFocus" >
           <bool>true</bool>
          </property>
          <property name="undoRedoEnabled" >
           <bool>false</bool>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
          <property name="tabStopWidth" >
           <number>79</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTabWidget" name="tabWidgetSource" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabSource1" >
           <attribute name="title" >
            <string>Plural 1</string>
           </attribute>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item row="0" column="0" >
             <widget class="QTextEdit" name="txtPluralSource1" />
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="lblComment" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>10</green>
            <blue>26</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>221</red>
            <green>223</green>
            <blue>228</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>85</green>
            <blue>85</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>199</red>
            <green>199</green>
            <blue>199</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>103</red>
            <green>141</green>
            <blue>178</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>238</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>82</red>
            <green>24</green>
            <blue>139</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>232</red>
            <green>232</green>
            <blue>232</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>10</green>
            <blue>26</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>221</red>
            <green>223</green>
            <blue>228</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>85</green>
            <blue>85</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>199</red>
            <green>199</green>
            <blue>199</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>103</red>
            <green>141</green>
            <blue>178</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>238</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>82</red>
            <green>24</green>
            <blue>139</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>232</red>
            <green>232</green>
            <blue>232</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>221</red>
            <green>223</green>
            <blue>228</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>85</green>
            <blue>85</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>199</red>
            <green>199</green>
            <blue>199</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>199</red>
            <green>199</green>
            <blue>199</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>86</red>
            <green>117</green>
            <blue>148</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>238</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>82</red>
            <green>24</green>
            <blue>139</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>232</red>
            <green>232</green>
            <blue>232</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip" >
       <string>Important Comment</string>
      </property>
      <property name="statusTip" >
       <string>Important Comment</string>
      </property>
      <property name="whatsThis" >
       <string>&lt;h3>Important Comment&lt;/h3>Hints from the developer to the translator are displayed in this area. This area will be hidden if there is no hint. </string>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QStackedWidget" name="targetStacked" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page3" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="txtTarget" >
          <property name="toolTip" >
           <string>Translated String</string>
          </property>
          <property name="statusTip" >
           <string/>
          </property>
          <property name="whatsThis" >
           <string>&lt;h3>Translated String&lt;/h3>This editor displays and lets you edit the translation of the currently displayed string.</string>
          </property>
          <property name="tabChangesFocus" >
           <bool>false</bool>
          </property>
          <property name="undoRedoEnabled" >
           <bool>false</bool>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
          <property name="tabStopWidth" >
           <number>79</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTabWidget" name="tabWidgetTarget" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabTarget1" >
           <attribute name="title" >
            <string>Plural 1</string>
           </attribute>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item row="0" column="0" >
             <widget class="QTextEdit" name="txtPluralTarget1" />
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QScrollBar" name="fileScrollBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Navigate in your file</string>
     </property>
     <property name="statusTip" >
      <string>Navigation Scrollbar</string>
     </property>
     <property name="whatsThis" >
      <string>&lt;h3>Navigation Scrollbar&lt;/h3>It allows you do navigate in the current file. If you filter your strings you get only the filtered list. &lt;br>It also gives you visual feedback about the postion of the current entry. The Tooltip also shows you the current number and the total numbers of strings.</string>
     </property>
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="tracking" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>588</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
